Currency.ts

```diff
@@ -397,6 +397,12 @@ export namespace Currency {
 	export function subtract(currency: Currency, minuend: number, subtrahend: number): number {
 		return round(round(minuend, currency) - round(subtrahend, currency), currency)
 	}
+	export function toMinor(amount: number, currency: Currency): number {
```

Comment:
Return undefined if decimalDigits for a currency is not defined rather than forcing 2. Force the implementor to decide instead what is the appropriate solution.

Reply:
Or add a parameter for a default value instead (defaultDecimals = 2)

Reply:
Changed so that currencies without decimalDefinitions are left as is.
E.g. 
`XPT` (platinum ounce, no decimals) will return the same amount as was send in.
This is not the same as `ISK` (islandic crown, with decimals=0), which will be rounded to whole units.